src/main/java/ru/rosatom/documentflow/dto/DocProcessCommentDto.java

```diff
@@ -0,0 +1,20 @@
+package ru.rosatom.documentflow.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@Schema(description = "Комментарий")
+public class DocProcessCommentDto {
+    @Schema(description = "Текст комментария")
+    private String textComment;
+    @Schema(description = "Автор комментария")
+    private UserReplyDto authorComment;
```

Comment:
Неправильный порядок слов
Приписка comment не нужна
```suggestion
    @Schema(description = "Автор комментария")
    private UserReplyDto author;
```